parts/k8s/kubernetesmastervars.t

```diff
@@ -181,7 +181,7 @@
   {{end}}
     "masterOffset": "[parameters('masterOffset')]",
 {{end}}
-    "apiVersionDefault": "2016-03-30",
+    "apiVersionDefault": "2018-04-01",
```

Comment:
What is the reason for that change? cc @jackfrancis we will need to do some back compat testing here

Reply:
Tests were failing with current value -> https://circleci.com/gh/Azure/acs-engine/27285

Reply:
In general we have been conservative about how and when API versions associated with different RPs are updated as new versions can cause bugs. Having said that new features will need new API versions so we can take in a new version but will need thorough testing.